vod_transcode.sh

```diff
@@ -9,6 +9,9 @@
 #  sudo service antmedia restart
 #
 
+# Just convert to HLS 
+HLS="1"
+
 # Don't forget to change the Ant Media Server App Name
 AMS_APP_NAME="WebRTCAppEE"
```

Comment:
I think no need to have folder name because server gives the full path of the file(So the app name is already in the variable.) It's likely that users will likely forget/ignore to change it.